src/tests/system/tests/test_default_debug_level.py

```diff
@@ -0,0 +1,180 @@
+from sssd_test_framework.topology import KnownTopology, KnownTopologyGroup
+
+
+@pytest.mark.topology(KnownTopology.Client)
+def test_default_debug_level__check(client: Client):
+    """
+    :title: Check default debug level when sssd started successfully
+    :setup:
+        1. Clear logs and cache
+        2. Start SSSD with default debug level
+    :steps:
+        1. Check log files
+    :expectedresults:
+        1. "Starting with debug level = 0x0070" is in each file
+            Log message with number "0x3f7c0" is stored, if log contains more than one line
+    :customerscenario: False
+    """
+    client.sssd.common.local()
+    client.sssd.default_domain = "local"
+
+    client.sssd.clear(db=True, memcache=True, logs=True, config=False)
+    client.sssd.start(debug_level=None)
+
+    for filename in ["sssd", "sssd_local", "sssd_nss", "sssd_pam"]:
+        log_str = client.fs.read(f"/var/log/sssd/{filename}.log")
+        assert "Starting with debug level = 0x0070" in log_str, f"Log file has wrong format: {log_str}"
+
+        if len(log_str.split("\n")) > 1:
+            assert "(0x3f7c0)" in log_str, f"Log file has wromg format: {log_str}"
+
+
+@pytest.mark.topology(KnownTopologyGroup.AnyProvider)
+def test_default_debug_level__check_with_login(client: Client, provider: GenericProvider):
+    """
+    :title: Successful login with default debug level doesn't generate any logs
+    :setup:
+        1. Add user to SSSD, set its password
+        2. Add fallback_homedir (generates extra logs on
+            user auth if not specified)
+        3. Clear cache and logs
+    :steps:
+        1. Start SSSD with default debug level
+        2. Check total log size before user authentication
+        3. Authenticate with user
+        4. Check total log size after authentication
+    :expectedresults:
+        1. SSSD started successfully
+        2. Checked correctly
+        3. User is authenticated
+        4. Log sizes before and after are same
+    :customerscenario: False
+    """
+
+    def log_length() -> int:
+        try:
+            output = client.fs.wc("/var/log/sssd/*", lines=True).stdout
+            return int(output.split()[0])
+        except SSHProcessError:
+            return 0
+
+    provider.user("user1").add(password="Secret123")
+    client.sssd.common.local()
+    client.sssd.default_domain = "local"
+    client.sssd.domain["fallback_homedir"] = "/home/%%u"
+
+    client.sssd.clear(db=True, memcache=True, logs=True, config=False)
+    client.sssd.start(debug_level=None)
+
+    start_log_length = log_length()
+    assert client.auth.ssh.password("user1", "Secret123"), "Authentication failed"
+    assert start_log_length == log_length(), "Debug messages were generated"
+
+
+@pytest.mark.ticket(bz=1893159)
+@pytest.mark.topology(KnownTopology.Client)
+def test_default_debug_level__fatal_and_critical_failures(client: Client):
+    """
+    :title: Check that messages with levels 0 and 1 are logged
```

Comment:
It seems like the sssd.conf permission change is an integral part of this test scenario, so I'd add it to the title. Based on the title for the next test case I'd propose: `Check that messages with levels 0 and 1 are logged when SSSD can't load the config file`

Reply:
I think that it is not that crucial part of the test. There could be anything else that generate these log level messages.

Reply:
Updated it, that it sounds better